app/destination/page.js

```diff
@@ -1,40 +1,86 @@
+import styles from "@/components/destination/destination.module.css";
+import { AddWishlistItem } from "@/components/destination/AddWishlistItem";
 
-// TASK - React 1 week 2
-// Move this to its own file
-const PlanetWishlistItem = ({
+const PlanetCard = ({
   name,
-  onRemove,
+  description,
   thumbnail,
+  isSelected,
+  onAddOrRemovePlanet,
 }) => {
   return (
-    <div className={styles.wishlistItem}>
-      <img className={styles.wishlistItemThumbnail} src={thumbnail} alt="" />
-      <b>{name.toUpperCase()}</b>
-      <button onClick={onRemove}>remove</button>
+    <div className={styles.planetCard}>
+      <img
+        className={styles.planetThumbnail}
+        src={thumbnail}
+        alt={`image of ${name}`}
+      />
+      <div className={styles.planetDescription}>
+        <h2>
+          {name}
+          {isSelected ? "- SELECTED" : ""}
+        </h2>
+        <p>{description}</p>
+      </div>
+      <button className="roundButton" onClick={() => onAddOrRemovePlanet(name)}>
+        {isSelected ? "REMOVE" : "ADD PLANET"}
+      </button>
     </div>
   );
-}
-
+};
 
 export const Destinations = () => {
-  const [selectedPlanets, onAddPlanet] = useState([]);
+  const planets = [
+    {
+      name: "Europa",
+      description:
+        "Europa, one of Jupiter’s moons, is an icy world with a hidden ocean beneath its surface. This mysterious moon is a prime candidate for the search for extraterrestrial life, making it a thrilling destination for space explorers.",
+      thumbnail: "/destination/image-europa.png",
+    },
+    {
+      name: "Mars",
+      description:
+        "Mars, the Red Planet, is a barren yet fascinating world with vast deserts, towering volcanoes, and the deepest canyon in the solar system. As humanity’s next frontier, Mars invites us to dream of colonization and the possibilities of life beyond Earth.",
+      thumbnail: "/destination/image-mars.png",
+      isSelected: true,
+    },
+    {
+      name: "Moon",
+      description:
+        "Our closest celestial neighbor, the Moon, is a silent witness to Earth's history. With its stunning craters and desolate landscapes, the Moon offers a unique glimpse into space exploration's past and future, making it a perfect destination for lunar adventurers.",
+      thumbnail: "/destination/image-moon.png",
+    },
+    {
+      name: "Titan",
+      description:
+        "Titan, Saturn's largest moon, is a world of dense atmosphere and liquid methane lakes. This enigmatic moon is shrouded in a thick orange haze, concealing a landscape that is both alien and strangely familiar, beckoning explorers to uncover its secrets.",
+      thumbnail: "/destination/image-titan.png",
+    },
+  ];
+  const [selectedPlanets, setSelectedPlanets] = useState([]);
 
   let isPlanetSelected = false;
-  let numberOfPlanets = 0;
+  let numberOfPlanets = selectedPlanets.length;
 
   const onAddOrRemovePlanet = (name, index) => {
```

Comment:
```suggestion
  const onAddOrRemovePlanet = (name) => {
```
You don't use the `index` in your function